server/odc-core/src/main/java/com/oceanbase/odc/core/authority/permission/ConnectionPermission.java

```diff
@@ -119,7 +119,7 @@ protected void initPermissionMask(int mask) {
 
     @Override
     public String toString() {
-        return "ConnectionPermission " + this.resourceType + ":" + this.resourceId + " " + getActions(this.mask);
+        return ResourceType.ODC_CONNECTION.getLocalizedMessage() + ":" + this.resourceId + ": " + getActions(this.mask);
```

Comment:
Where does this copywriting come into play?

Reply:
`com.oceanbase.odc.core.authority.exception.AccessDeniedException`